### ISSUE TITLE
Slashed validator without `EFFECTIVE_BALANCE/32` initial penalty report?

### ISSUE THREAD

**Issue by nalepae** [gobitfly/eth2-beaconchain-explorer#2322]
Validator `48608` has been slashed: https://beaconcha.in/validator/48608

```
curl http://localhost:5052/eth/v1/beacon/states/head/validators/48609 | jq
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   488  100   488    0     0   4722      0 --:--:-- --:--:-- --:--:--  4880
{
  "execution_optimistic": false,
  "data": {
    "index": "48609",
    "balance": "35126907760",
    "status": "exited_slashed",
    "validator": {
      "pubkey": "0xa26deb28799034075f265009f2a5c696e066be48ff36d14a0de761a200a2540569b592412932aefe493b6aa69fc038e2",
      "withdrawal_credentials": "0x006209a113a4d8bff906b882439a8a6c1d159f87ba9eabfc8b31501b64bca53e",
      "effective_balance": "32000000000",
      "slashed": true,
      "activation_eligibility_epoch": "4066",
      "activation_epoch": "6931",
      "exit_epoch": "200747",
      "withdrawable_epoch": "208934"
    }
  }
}
```

However, we cannot see in "Charts" the initial `EFFECTIVE_BALANCE/32` penalty (in its case: 1ETH)

![image](https://github.com/gobitfly/eth2-beaconchain-explorer/assets/4943830/0a1cf44f-3209-438b-8009-4b2771581fcd)
(No block proposal this day, neither sync committee, neither top-up deposit that could "hide" the initial slashing penalty)

### Extra data:

```
➜  ~ curl http://localhost:5052/eth/v1/beacon/states/6424518/validator_balances\?id\=48608 | jq
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100    81  100    81    0     0     92      0 --:--:-- --:--:-- --:--:--    92
{
  "execution_optimistic": false,
  "data": [
    {
      "index": "48608",
      "balance": "35296293181"
    }
  ]
}
➜  ~ curl http://localhost:5052/eth/v1/beacon/states/6424517/validator_balances\?id\=48608 | jq
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100    81  100    81    0     0     92      0 --:--:-- --:--:-- --:--:--    93
{
  "execution_optimistic": false,
  "data": [
    {
      "index": "48608",
      "balance": "36296293181"
    }
  ]
}
```

**Comment by recy21** [comment]
Thank you for your interest in our project.
Created an internal ticket, will update here if we have new information.

**Comment by ALIR86** [comment]
Why consensual is zero ETH #daily income

**Comment by VladimirGabriel** [comment]
> O validador `48608`foi cortado: https://beaconcha.in/validator/48608
> 
> ```
> curl http://localhost:5052/eth/v1/beacon/states/head/validators/48609 | jq
>   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
>                                  Dload  Upload   Total   Spent    Left  Speed
> 100   488  100   488    0     0   4722      0 --:--:-- --:--:-- --:--:--  4880
> {
>   "execution_optimistic": false,
>   "data": {
>     "index": "48609",
>     "balance": "35126907760",
>     "status": "exited_slashed",
>     "validator": {
>       "pubkey": "0xa26deb28799034075f265009f2a5c696e066be48ff36d14a0de761a200a2540569b592412932aefe493b6aa69fc038e2",
>       "withdrawal_credentials": "0x006209a113a4d8bff906b882439a8a6c1d159f87ba9eabfc8b31501b64bca53e",
>       "effective_balance": "32000000000",
>       "slashed": true,
>       "activation_eligibility_epoch": "4066",
>       "activation_epoch": "6931",
>       "exit_epoch": "200747",
>       "withdrawable_epoch": "208934"
>     }
>   }
> }
> ```
> 
> Porém, não podemos ver nos “Gráficos” a `EFFECTIVE_BALANCE/32`penalidade inicial (no seu caso: 1ETH)
> 
> ![imagem](https://user-images.githubusercontent.com/4943830/246448901-0a1cf44f-3209-438b-8009-4b2771581fcd.png) (Nenhuma proposta de bloqueio neste dia, nem comitê de sincronização, nem depósito de recarga que pudesse "ocultar" a penalidade inicial)
> 
> ### Dados extras:
> ```
> ➜  ~ curl http://localhost:5052/eth/v1/beacon/states/6424518/validator_balances\?id\=48608 | jq
>   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
>                                  Dload  Upload   Total   Spent    Left  Speed
> 100    81  100    81    0     0     92      0 --:--:-- --:--:-- --:--:--    92
> {
>   "execution_optimistic": false,
>   "data": [
>     {
>       "index": "48608",
>       "balance": "35296293181"
>     }
>   ]
> }
> ➜  ~ curl http://localhost:5052/eth/v1/beacon/states/6424517/validator_balances\?id\=48608 | jq
>   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
>                                  Dload  Upload   Total   Spent    Left  Speed
> 100    81  100    81    0     0     92      0 --:--:-- --:--:-- --:--:--    93
> {
>   "execution_optimistic": false,
>   "data": [
>     {
>       "index": "48608",
>       "balance": "36296293181"
>     }
>   ]
> }
> ```

**Comment by VladimirGabriel** [comment]
https://github.com/gobitfly/eth2-beaconchain-explorer/issues/2322#issue-1760781696

**Comment by recy21** [comment]
Fixed in [v1.31.0](https://github.com/gobitfly/eth2-beaconchain-explorer/releases/tag/v1.31.0)
